docs/usage.md

```diff
@@ -392,16 +392,46 @@ the code integration is very simple: a single call to `bristlemouth_init`.
 
 ```C
 #include "bristlemouth.h"
+#include "l2.h"
 
 void network_device_power_callback(bool on) {
     // Provide or cut off power to the network chip.
     // On the mote, we set pin PH1 high or low.
 }
 
+static void network_device_interrupt(void) {
+    // Necessary logic that occurs when an interrupt from the 10Base-T1L driver
+    // is called should be placed here (application specific), the below
+    // function call is a must in order to inform L2 that an event must be
+    // handled
+    bm_l2_handle_device_interrupt();
```

Comment:
It looks like this should change to `bristlemouth_handle_network_device_interrupt` which would allow us to remove the l2.h include above, right?